cmd/cloud/cluster.go

```diff
@@ -45,6 +49,19 @@ var clusterCreateCmd = &cobra.Command{
 	},
 }
 
+var clusterUpgradeCmd = &cobra.Command{
+	Use:   "upgrade",
+	Short: "Upgrade a cluster.",
```

Comment:
Wonder if we should explicitly call out that this upgrades Kubernetes -- might not be clear what is actually being "upgraded" as such:
```suggestion
	Short: "Upgrade k8s on a cluster.",
```

Reply:
For sure, I meant to do that before I pushed this up.